recipes-extended/pam/libpam_1.%.bbappend

```diff
@@ -1 +1,23 @@
 FILESEXTRAPATHS:prepend := "${THISDIR}/${BPN}:"
+
+SRC_URI += "\
+	file://security/faillock.conf \
+"
+
+do_install:append() {
+	install -m 644 ${WORKDIR}/security/faillock.conf ${D}${sysconfdir}/security/faillock.conf
+}
+
+pkg_postinst:pam-plugin-faillock:append() {
+	# enable faillock
+	sed -E -i 's/^(.+)success=1(.+)$/auth    requisite pam_faillock.so preauth\n\1success=2\2\nauth    [default=die] pam_faillock.so authfail/' "${sysconfdir}/pam.d/common-auth"
+	echo "auth    sufficient pam_faillock.so authsucc" >> "${sysconfdir}/pam.d/common-auth"
+}
+
+pkg_postrm:pam-plugin-faillock:append() {
```

Comment:
`postinst` steps are normally undone in the package `prerm` script. Otherwise, you might encounter an issue removing the package files and end up in a state where the common-auth script rules pass into non-existant PAM libraries.

Reply:
Fixed. FWIW, I used pkg_postrm because that's what I saw in [screen](https://github.com/ni/openembedded-core/blob/2566f8e51d56848d8b28f37462160e90253b79fc/meta/recipes-extended/screen/screen_4.9.1.bb#L47).